react/README.md

```diff
@@ -279,6 +279,22 @@
   <div />
   ```
 
+- Avoid using array index as `key` prop. ([why?](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318))
+
+```jsx
```

Comment:
The alignment is off compared to the other items in the list. I think the way they keep it indented is to add a `> ` block before the code sample.

![screen shot 2016-05-25 at 12 11 29 pm](https://cloud.githubusercontent.com/assets/133937/15553011/f49e1d92-2271-11e6-8cf8-3559663a0472.png)